### ISSUE TITLE
Can’t comment on issue

### ISSUE THREAD

**Issue by Aareon** [GitHawkApp/GitHawk#2224]
If I try to comment on any issue in any repo,  my message doesn’t post. The message is kept in the text box, I’m sure for safe keeping. I’ve only seen the “Something went wrong” error message a couple out of many times I’ve tried.

<details>
<summary>Bug Report Dump (Auto-generated)</summary>
<pre>
Version 1.22.0 (1534395229)
Device: iPhone 7 (iOS 12.0)
TestFlight: false
</pre>
</details>

**Comment by Aareon** [comment]
It seems that if I have the message and keyboard up when the error happens, the “Something went wrong” message is just hidden behind it. When the “SWW” message *does* appear, I can’t select, pull up, or do anything other than acknowledge it.

**Comment by Aareon** [comment]
Strangely enough, I *can* comment here, though...

**Comment by j-f1** [comment]
This seems to be an issue with organizations not allowing the app to post comments. You can work around the issue by creating and logging in with a personal access token.

**Comment by rnystrom** [comment]
@Aareon can you let us know the repo? Is it used by an organization? It might need to remove restrictions for GitHawk.

**Comment by rnystrom** [comment]
Agree, error messaging could be improved throughout the whole app.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
